lib/iris/analysis/__init__.py

```diff
@@ -1641,7 +1641,9 @@ def interp_order(length):
 
 * weights (float ndarray):
     Weights matching the shape of the cube, or the length of
-    the window for rolling window operations.
+    the window for rolling window operations. Weights should be
+    normalized before passing them into this function if scaling
```

Comment:
"passing them into this function" ... it's not actually used like a function, so in the spirit of avoiding confusion, how about "using them with this aggregator" instead?